clients/gwtrpc/src/main/java/org/activityinfo/legacy/shared/command/CancelDatabaseTransfer.java

```diff
@@ -0,0 +1,23 @@
+package org.activityinfo.legacy.shared.command;
+
+import org.activityinfo.legacy.shared.command.result.VoidResult;
+
+public class CancelDatabaseTransfer implements Command<VoidResult> {
```

Comment:
I would really like to avoid adding new RPC commands as this only adds to our technical debt of endpoints to be migrated later. Can you add these instead to the DatabaseResource? I'd suggest:
```
POST /resources/database/{databaseId}/startTransfer
POST /resource/database/{databaseId}/cancelTransfer
```

You can add a model object in `org.activityinfo.model.database` for the request and response body.

For the email links, all we really need is the token and decision, so maybe something like:

```
GET /confirm/approve?token={transferToken}
GET /confirm/reject?token={transferToken}
```

Then we can start building out more here later for confirming/rejecting invitations to ActivityInfo in general....

Reply:
Sure. I leaned heavily on the RPC commands actually, probably because of familiarity with the solution/following some of the current code. I'll move all this to the DatabaseResource and out of the RPCs.

Totally agree with the accept/reject endpoint. I had some ideas on basically this before I got caught up in pipe(line) dreams. I'll use some of that here so we can make it extensible for the future.